packages/create-schemas/src/openapiTypescriptHelper.ts

```diff
@@ -1,11 +1,24 @@
-import openapiTS, { astToString, type OpenAPITSOptions } from "openapi-typescript";
-import { generateExportEndpointsTypeDeclaration, generateExportSchemaTypeDeclaration, getSchemaNames } from "./astHelper.ts";
-
-export async function generateSchemas(openApiPath: string, outputPath: string, openApiTsOptions: OpenAPITSOptions): Promise<string> {
-    const CWD = new URL(`file://${process.cwd()}/`);
+import openapiTS, { astToString } from "openapi-typescript";
+import {
+    generateExportEndpointsTypeDeclaration,
+    generateExportSchemaTypeDeclaration,
+    getSchemaNames
+} from "./astHelper.ts";
+import type { ResolvedConfig } from "./config.ts";
+import { pathToFileURL } from "url";
+import { join } from "path";
+
+export async function generateSchemas(config: ResolvedConfig): Promise<string> {
+    const url = URL.canParse(config.input)
+        ? config.input
+        : pathToFileURL(join(config.root, config.input));
```

Comment:
Gotta fix this, it's a bit jank and doesn,t cover all combinations (input as file url, input as file path, root as file url, etc.)